Wox.Infrastructure/UserSettings/Settings.cs

```diff
@@ -49,6 +49,8 @@ public class Settings : BaseModel
         public bool IgnoreHotkeysOnFullscreen { get; set; }
 
         public HttpProxy Proxy { get; set; } = new HttpProxy();
+
+        public Action Save = ()=> {};
```

Comment:
I think at first I added [JsonIgnore] and then removed as wanted to have least amount of code impact. Feel free to add it.